calculator/src/main/java/edu/nextstep/camp/calculator/CalculatorActivity.kt

```diff
@@ -22,10 +23,31 @@ class CalculatorActivity : AppCompatActivity() {
             viewModel = calculatorVM
         }
         setContentView(binding.root)
+        initView()
 
+        observeExpressionHistory()
         observeIncompleteExpressionErrorEvent()
     }
 
+    private fun initView() {
+        initRecyclerView()
+    }
+
+    private fun initRecyclerView() {
+        binding.recyclerView.apply {
+            adapter = calculatorHistoryAdapter
+        }
```

Comment:
```suggestion
        binding.recyclerView.adapter = calculatorHistoryAdapter
```
apply 범위연산자를 사용하지 않는게 더 간결해보이네요. 🤔

Reply:
[35bbe9d](https://github.com/next-step/android-calculator-mvvm/pull/113/commits/35bbe9dfd8057d77fa45fa5330b128b678b90bb8)

넵넵 감사합니다
현석님께 한가지 질문이 있다면 혹시 현석님은 범위연산자를 지향하시는 편이신가요 지양하시는 편이신가요?
물론 가독성을 높혀주기 위해 필요해 보이는 경우가 명백히 있지만, 범위 연산자에대한 사용이 각 회사마다 달라서 지양한다는 글을 본것 같아 질문드립니다~

Reply:
범위연산자를 지양하지는 않습니다만, 주로 둘 이상의 값을 설정해야할 경우에 범위연산자를 사용합니다.